core/src/org/pentaho/di/core/row/value/ValueMetaBase.java

```diff
@@ -80,27 +80,45 @@
 
 public class ValueMetaBase implements ValueMetaInterface {
 
-  // region ValueMetaBase Attributes
-  protected static Class<?> PKG = Const.class; // for i18n purposes, needed by Translator2
+  // region Default Numeric Types Parse Format
+  private static final String DEFAULT_INTEGER_PARSE_MASK = "####0";
+  private static final String DEFAULT_NUMBER_PARSE_MASK = "####0.0#########";
+  private static final String DEFAULT_BIGNUMBER_PARSE_MASK = "######0.0###################";
+
+  private static final String DEFAULT_DATE_PARSE_MASK = "yyyy/MM/dd HH:mm:ss.SSS";
+  private static final String DEFAULT_TIMESTAMP_PARSE_MASK = "yyyy/MM/dd HH:mm:ss.SSSSSSSSS";
+  // endregion
+
+  // region Default Types Format
+  static final String DEFAULT_INTEGER_FORMAT_MASK = Const.NVL(
+          EnvUtil.getSystemProperty( Const.KETTLE_DEFAULT_INTEGER_FORMAT ),
+          "####0;-####0" );
 
-  static final String DEFAULT_INTEGER_FORMAT_MASK = " ###############0;-###############0";
+  static final String DEFAULT_NUMBER_FORMAT_MASK = Const.NVL(
+          EnvUtil.getSystemProperty( Const.KETTLE_DEFAULT_NUMBER_FORMAT ),
+          "####0.0#########;-####0.0#########" );
 
-  //TODO: this value was extracted from the code. Does it make sense to have the same mask for this types?
-  static final String DEFAULT_NUMBER_FORMAT_MASK = " ##########0.0########;-#########0.0########";
-  static final String DEFAULT_BIG_NUMBER_FORMAT_MASK = " ##########0.0########;-#########0.0########";
+  static final String DEFAULT_BIG_NUMBER_FORMAT_MASK = Const.NVL(
+          EnvUtil.getSystemProperty( Const.KETTLE_DEFAULT_BIGNUMBER_FORMAT ),
+          "######0.0###################;-######0.0###################" );
 
-  public static final String DEFAULT_DATE_FORMAT_MASK = Const.NVL( EnvUtil
-      .getSystemProperty( Const.KETTLE_DEFAULT_DATE_FORMAT ), "yyyy/MM/dd HH:mm:ss.SSS" );
+  static final String DEFAULT_DATE_FORMAT_MASK = Const.NVL( EnvUtil
+          .getSystemProperty( Const.KETTLE_DEFAULT_DATE_FORMAT ), "yyyy/MM/dd HH:mm:ss.SSS" );
 
-  public static final String DEFAULT_TIMESTAMP_FORMAT_MASK = Const.NVL( EnvUtil
-      .getSystemProperty( Const.KETTLE_DEFAULT_TIMESTAMP_FORMAT ), "yyyy/MM/dd HH:mm:ss.SSSSSSSSS" );
+  static final String DEFAULT_TIMESTAMP_FORMAT_MASK = Const.NVL( EnvUtil
```

Comment:
@TatsianaKasiankova Thanks. It shows how important your review was... as it did break the QAT. :-(

It's been fixed by https://github.com/pentaho/pentaho-kettle/pull/3651.